src/java/picard/fingerprint/CheckFingerprint.java

```diff
@@ -64,9 +66,14 @@
     @Option(shortName=StandardOptionDefinitions.INPUT_SHORT_NAME, doc = "Input SAM or BAM file.")
     public File INPUT;
 
-    @Option(shortName=StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "Base of output files to write.")
+    @Option(shortName=StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "If SUMMARY_OUTPUT is null, the base of output files to write, " +
+            "otherwise the full path to the detailed metrics.")
     public String OUTPUT;
 
+    @Option(shortName = "S", doc = "The text file to which to write summary metrics.  If this is null, then OUTPUT is the base name of " +
+            "the output files and the default suffixes will be used.", optional = true)
```

Comment:
can you point out what the suffixes are in the doc so that the user will not have to read the code?